media/src/main/kotlin/com/imashnake/animite/media/MediaPage.kt

```diff
@@ -397,8 +393,6 @@ fun MediaCharacters(
             image = character.image,
             label = character.name,
             onClick = { Log.d("CharacterId", "${character.id}") },
-            imageHeight = dimensionResource(R.dimen.character_image_height),
-            cardWidth = dimensionResource(R.dimen.character_card_width),
```

Comment:
Won't removing this break it since we'll then be using the media card dimens for character cards by default?

Reply:
Oops didn't realise they were different, IDE collapses them by default 😓